.github/workflows/release.yaml

```diff
@@ -58,6 +74,7 @@ jobs:
     # TODO: currently this doesn't publish it as an artifact; would be a welcome contribution
     # to add that (and eventually as a release asset)
     runs-on: ubuntu-latest
+    if: false
```

Comment:
Why do we disable this & the rpm? Even if we're not publishing it yet, it's still useful to build, and is an intermediate step to publishing, no?

Reply:
> Why do we disable this & the rpm? Even if we're not publishing it yet, it's still useful to build, and is an intermediate step to publishing, no?

In my opinion these jobs need to be rewritten as the package needs to be created based on the uploaded prqlc.
Currently they are building prqlc from scratch, which takes extra time.

Reply:
But then why not run on releases only?

By turning off tests, it'll decay, and we'll be further from getting it publishing. Generally stuff that isn't tested breaks! And then contributors are less likely to contribute smaller changes, since their work won't be kept current.

Computing time is so much cheaper than contributor time! So anything we can do to make contributing easier is likely to be worthwhile...

(A future version could run nightly rather than on every merge and then run these, but that's another project)

Reply:
I enabled when not dry run.